include/TrustWalletCore/TWBravoAddress.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "TWBase.h"
+#include "TWData.h"
+#include "TWString.h"
+
+#include "TWBravoAddressType.h"
+
+TW_EXTERN_C_BEGIN
+
+struct TWPublicKey;
+
+/// Represents a Bravo address.
+TW_EXPORT_CLASS
+struct TWBravoAddress;
```

Comment:
from my understanding, this is just public key? We might need an `Account` for eos like chains

Reply:
The `Account` name can be validated using rpc call `get_account`. This will also return all associated public keys. 
Unlike most blockchains, Bravo requires user to register first. Registration requires 4 keypairs: “owner”, “active”, “posting”, “memo” and a unique username. Transferring the currency only requires the username and the associated “active” key. Hence, an application using the wallet-core library must ask user for their username and private key (WIF Format) during enrollment. After this, the app should check if the private key is indeed the username’s active key by send the RPC call “get_accounts” to bravo node. The will give the active public key along with other account information. You need to compare this address with the address you derive from private key and ensure they match.